src/Admin/Product/Attributes/AttributesTab.php

```diff
@@ -193,7 +193,7 @@ protected function get_applicable_product_types(): array {
 	 * @return array of WooCommerce product types (e.g. 'subscription', 'variable-subscription', etc.)
 	 */
 	protected function get_hidden_product_types(): array {
-		return apply_filters( 'wc_pinterest_attributes_tab_hidden_product_types', array( 'subscription', 'variable-subscription' ) );
+		return apply_filters( 'wc_pinterest_attributes_tab_hidden_product_types', array_diff( array_keys( wc_get_product_types() ), $this->get_applicable_product_types() ) );
```

Comment:
Along with the previous comment, we should split this across multiple lines to make it a bit more readable.